CHANGELOG.md

```diff
@@ -1,5 +1,83 @@
 <!-- if you're not the release manager, do your edits to changelog under CHANGELOG.d/ -->
 
+# [2022-01-18]
+
+## Release notes
+
+* This release introduces a mandatory `federationDomain` configuration setting to cargohold. Please update your `values/wire-server/values.yaml` to set `cargohold.settings.federationDomain` to the same value as the corresponding option in galley (and brig). (#1990)
+* The brig server config option `setDefaultLocale` has been replaced by `setDefaultUserLocale` and `setDefaultTemplateLocale` (see docs/reference/config-options.md for details) (#2028)
+* From this release onwards, the images for haskell components (brig, galley,
+  cargohold, etc.) will be using Ubuntu 20.04 as the base. The images are about
+  30-35 MB larger than the previous alpine based images. (#1852)
+* Wire cloud operators: Make sure [#35](https://github.com/wireapp/ansible-sft/pull/35) is applied to all SFT servers before deploying. (#2030)
+
+## API changes
+
+* The deprecated endpoint `GET /teams` now ignores query parameters `ids`, `start` (#2027)
+* Add qualified v4 endpoints for downloading and deleting assets. The upload API is still on the same path, but the asset object it returns now contains a `domain` field. Note that federated behaviour is still not implemented. (#2002)
+* Enable downloading assets from a remote (federated) cargohold instance via the v4 API.
+  The content of remote assets is returned as stream with content type
+  `application/octet-stream`.
+  Please refer to the Swagger API documentation for more details. (#2004)
```

Comment:
These sound like they could be merged into one line (together with the asset bugfix 2037 and 2038). The problem with the automated changelog is than the resulting changelog is of less good quality as multiple PRs together implementing one thing are not grouped into one changelog item, but many individual ones.

Not a blocker though. Some other things in this changelog could also be combined. (SFT stuff, a range of internal items)

Reply:
It's not really a problem with the automated changelog, just a mistake. Whoever wrote the second entry should have updated the first one as well. This is going to be true for any system where the changelog is updated incrementally at every commit. But honestly, I'm not sure why we're not just using the PR description here.